src/Types/ForceReply.php

```diff
@@ -52,9 +60,10 @@ class ForceReply extends BaseType
      * @param bool $forceReply
      * @param bool|null $selective
      */
-    public function __construct($forceReply = true, $selective = null)
+    public function __construct($forceReply = true, $inputFieldPlaceholder = false, $selective = null)
```

Comment:
Please move`$inputFieldPlaceholder` to the last position as it would be a BC brake

Reply:
Sure, It is done